examples/platform/nrfconnect/util/DFUOverSMP.cpp

```diff
@@ -130,8 +134,21 @@ void DFUOverSMP::StartBLEAdvertising()
     }
 }
 
+void DFUOverSMP::OnBleConnect(bt_conn * conn, uint8_t err)
+{
+    if (GetDFUOverSMP().IsEnabled())
+    {
+        (void) k_work_submit(&sDFUOverSMP.mFlashWakeUpWork);
```

Comment:
is the cast required? The docs I have say the method is already void:

https://docs.zephyrproject.org/1.9.0/api/kernel_api.html#_CPPv313k_work_submitP6k_work

Reply:
Yes, it has changed in the newer zephyr: https://docs.zephyrproject.org/latest/kernel/services/threads/workqueue.html?highlight=k_work_submit#c.k_work_submit.